applications/vessel-history/src/features/profile/components/activity/Activity.tsx

```diff
@@ -56,7 +62,7 @@ const Activity: React.FC<InfoProps> = (props): React.ReactElement => {
                         <I18nDate date={group.entries[group.entries.length - 1].end} format={DateTime.DATE_SHORT} />
```

Comment:
Start and end time of the group should be set in the state too

Reply:
not understand this